Src/IronPython.Modules/nt.cs

```diff
@@ -1805,6 +1979,32 @@ public static void utime(string path, [ParamDictionary]IDictionary<string, objec
         }
 
 #endif
+        private static string ToFsString(this Bytes b) => _filesystemEncoding.GetString(b.AsMemory().Span);
+
+        private static Bytes ToFsBytes(this string s) => Bytes.Make(_filesystemEncoding.GetBytes(s));
+
+        private static Bytes ToFsBytes(this IBufferProtocol bp) {
+            // TODO: DeprecationWarning starting from Python 3.7
```

Comment:
I would be fine with adding the deprecation warning right away (if it doesn't cause testing failures).